package.json

```diff
@@ -82,13 +81,12 @@
     "dayjs": "^1.11.7",
     "dotenv": "16.0.3",
     "enketo-core": "^5.16.6",
-    "eslint": "8.57.0",
-    "eslint-config-prettier": "10.1.1",
+    "eslint": "9.27.0",
     "eslint-plugin-cypress": "2.13.2",
```

Comment:
Why didn't you upgrade `eslint-plugin-cypress` but upgraded all other ESLint plugins?